Wikipedia/Code/PreviewAndSaveViewController.m

```diff
@@ -122,13 +122,21 @@ - (BOOL)prefersStatusBarHidden {
 }
 
 - (void)wmf_showAlertForTappedAnchorHref:(NSString *)href {
+    
     UIAlertController *alertController =
-        [UIAlertController alertControllerWithTitle:href
-                                            message:nil
+        [UIAlertController alertControllerWithTitle:@"Your changes have not been saved yet"
```

Comment:
@anirudh24seven would you mind adding these strings to the english locatable.strings file? 
Also, there is a locatable.strings file for "qqq" - in that file we add a description to the string so translators know the context. Just open it up and you will see what I mean. 

After that you can use the `MWLocalizedString` macro to reference those strings his this file.

Let me know if try to do this and run into issues and thanks for the contribution!